test/threading.cpp

```diff
@@ -346,6 +346,7 @@ TEST(Threading, FFT_R2C)
 
     int numDevices = 1;
```

Comment:
Wouldn't it be better to initialize `numDevices` to 0?

Reply:
Probably, I didn't touch existing declaration. It wouldn't matter anyway, since af_get_device_count is going to overwrite it with 0 or available device count.

Reply:
This should be set to -1. If the af_get_device_count fails it will most likely not change the value.